Deep_learning/Deep-learning.rst

```diff
@@ -1095,29 +1061,3 @@ Release Notes
     * Protobuf - `V3.5.2 <https://github.com/protocolbuffers/protobuf/releases/tag/v3.5.2>`_
```

Comment:
Check with chris austen about MIGraphX, MIVisionX

Reply:
there is nothing about migraphx in this file.  MIVisionX is owned by... 
kiriti.nageshgowda@amd.com

Rajy.MeeyakhanRawther@amd.com